src/javascript/EditPanel/EditPanelContent/FormBuilder/FormBuilder.jsx

```diff
@@ -19,7 +19,12 @@ const FormBuilder = ({mode}) => {
         return <></>;
     }
 
-    const isOrderingSection = !nodeData.isSite && !nodeData.isPage && nodeData.primaryNodeType.hasOrderableChildNodes && mode === Constants.routes.baseEditRoute;
+    const isAutomaticallyOrderable = nodeData.primaryNodeType.supertypes.find(s => s.name === Constants.automaticOrdering.mixinAutomaticallyOrderable);
```

Comment:
Why not reuse the same test: 
**const canAutomaticOrder = values && values[Constants.automaticOrdering.mixin] !== undefined;**
done in ChildrenSectionCmp ?
There is no need for specific check on jmix:automaticallyOrderable, jmix:orderedList is the extends that provide the fieldset, and it's extending jmix:automaticallyOrderable.
So the check **values && values[Constants.automaticOrdering.mixin] !== undefined;** should be enough.